Jenkinsfile

```diff
@@ -63,5 +76,5 @@ def buildManifest = {String manifest, String bitbake_image ->
 parallel 'core':{
     node("DockerCI") { buildManifest("pelux-intel.xml", "core-image-pelux") }
 },'qtauto':{
-    node("DockerCI") { buildManifest("pelux-intel-qt.xml", "core-image-pelux-qt") }
+    //node("DockerCI") { buildManifest("pelux-intel-qt.xml", "core-image-pelux-qt") }
```

Comment:
Looks good but it seems like you have commented out a line here. Is there a reason behind this?